pkg/cstor/cstor.go

```diff
@@ -536,3 +506,16 @@ func (p *Plugin) getVolInfo(volumeID, snapName string) (*Volume, error) {
 
 	return vol, nil
 }
+
+// getScheduleName return the schedule name for the given backup
+func (p *Plugin) getScheduleName(backupName string) string {
+	// for non-scheduled backup, we are considering backup name as schedule name only
+	scheduleName := backupName
```

Comment:
This answer the question I had on the related Maya PR. It is better to rename the parameter as scheduleOrBackupName to be explicit.

Reply:
ok